src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -1089,4 +1125,33 @@ export default class MutationTable<
             />
         );
     }
+
+    protected getMutations() {
```

Comment:
this should be computed getter since it is a computation of props.

Reply:
We wouldn't need this function if `anyMutationHasASCNProperty` was handled upstream in a store. See my comment below.

Reply:
[comment for reference](https://github.com/cBioPortal/cbioportal-frontend/pull/2762#discussion_r426906350)